main/src/main/scala/sbt/Defaults.scala

```diff
@@ -541,20 +541,24 @@ object Defaults extends BuildCommon {
     }
   }
 
-  def allTestGroupsTask(s: TaskStreams, frameworks: Map[TestFramework, Framework], loader: ClassLoader, groups: Seq[Tests.Group], config: Tests.Execution, cp: Classpath, javaHome: Option[File]): Task[Tests.Output] = {
-    allTestGroupsTask(s, frameworks, loader, groups, config, cp, javaHome, forkedParallelExecution = false)
+  def allTestGroupsTask(s: TaskStreams, frameworks: Map[TestFramework, Framework], loader: ClassLoader, groups: Seq[Tests.Group], config: Tests.Execution, cp: Classpath, javaHome: Option[File], javaOptions: Seq[String]): Task[Tests.Output] = {
+    allTestGroupsTask(s, frameworks, loader, groups, config, cp, javaHome, javaOptions, forkedParallelExecution = false)
   }
 
-  def allTestGroupsTask(s: TaskStreams, frameworks: Map[TestFramework, Framework], loader: ClassLoader, groups: Seq[Tests.Group], config: Tests.Execution, cp: Classpath, javaHome: Option[File], forkedParallelExecution: Boolean): Task[Tests.Output] = {
+  def allTestGroupsTask(s: TaskStreams, frameworks: Map[TestFramework, Framework], loader: ClassLoader, groups: Seq[Tests.Group], config: Tests.Execution, cp: Classpath, javaHome: Option[File], javaOptions: Seq[String], forkedParallelExecution: Boolean): Task[Tests.Output] = {
```

Comment:
See below, this is what I do when the `runPolicy` is `SubProcess`, as I have access to the `ForkOptions`.
Unfortunately, I cannot do that when the Strategy is `InProcess`...